src/main/java/com/exasol/exasoltestsetup/standalone/StandaloneExasolTestSetup.java

```diff
@@ -181,7 +182,10 @@ public SqlConnectionInfo getConnectionInfo() {
     public Bucket getDefaultBucket() {
         return SyncAwareBucket.builder().ipAddress("localhost").port(this.localBucketFsPort).name("default")
                 .serviceName("bfsdefault").readPassword(this.bucketFsReadPassword)
-                .writePassword(this.bucketFsWritePassword).monitor(new WaitBucketFsMonitor()).build();
+                .writePassword(this.bucketFsWritePassword) //
+                .monitor(new WaitBucketFsMonitor()) //
+                .stateRetriever(new TimestampRetriever()) //
```

Comment:
This line was missing and caused a nullpointer exception ;)

Reply:
the `.build()` was missing, right?

Reply:
OK, the stateRetriever! Damn!
Yes, 1000 thanks for the hint to add a check to the constructor!